BTCPayServer.Plugins.USDt/Configuration/Ethereum/EthUSDtLikeConfigurationItem.cs

```diff
@@ -0,0 +1,18 @@
+using System;
+
+namespace BTCPayServer.Plugins.USDt.Configuration.Ethereum;
+
+public record EthUSDtLikeConfigurationItem : USDtPluginConfigurationItem
```

Comment:
I think this one is not used. 
However, I think this is good one in term of naming. Indeed, this plugin aims to cover only USDt for now, it could support TRC20/ERC20 though.